website/src/libs/api-v3/i18n.js

```diff
@@ -0,0 +1,98 @@
+import fs from 'fs';
+import path from 'path';
+import _ from 'lodash';
+import shared from '../../../../common';
+
+export const localePath = path.join(__dirname, '/../../../../common/locales/');
+
+// Store translations
+export let translations = {};
+// Store MomentJS localization files
+export let momentLangs = {};
+
+// Handle differencies in language codes between MomentJS and /locales
+let momentLangsMapping = {
+  en: 'en-gb',
+  en_GB: 'en-gb', // eslint-disable-line camelcase
+  no: 'nn',
+  zh: 'zh-cn',
+  es_419: 'es', // eslint-disable-line camelcase
+};
+
+function _loadTranslations (locale) {
+  let files = fs.readdirSync(path.join(localePath, locale));
+
+  translations[locale] = {};
+
+  files.forEach((file) => {
+    if (path.extname(file) !== '.json') return;
+
+    // We use require to load and parse a JSON file
+    _.merge(translations[locale], require(path.join(localePath, locale, file))); // eslint-disable-line global-require
+  });
+}
+
+// First fetch English strings so we can merge them with missing strings in other languages
+_loadTranslations('en');
+
+// Then load all other languages
+fs.readdirSync(localePath).forEach((file) => {
+  if (file === 'en' || fs.statSync(path.join(localePath, file)).isDirectory() === false) return;
+  _loadTranslations(file);
+
+  // Merge missing strings from english
+  _.defaults(translations[file], translations.en);
+});
+
+// Add translations to shared
+shared.i18n.translations = translations;
```

Comment:
It was this way in the old code, probably there's a reason for it, the first one that comes to my mind is that the compiling step for the client would load a lot more data and also doesn't support the fs module